Assets/Scripts/Player/Body/Body.cs

```diff
@@ -14,10 +14,14 @@ public class Body : NetworkBehaviour
 	{
 		///The BodyPart objects that this MonoBehaviour is responsible for managing
 		[SerializeField] private List<BodyPart> bodyParts = new List<BodyPart>();
```

Comment:
If you aren't using this in the inspector then we might as well not have `[SerializeField]` on there

Reply:
It was being used, I'll change now as I automated it